### ISSUE TITLE
Using OomBootstrapper in console/horizon/queue context

### ISSUE THREAD

**Issue by ragingdave** [bugsnag/bugsnag-laravel#440]
### Description
So in looking to utilize this in a non-http context, I took a stab at just using the OomBootstrapper in the Console Kernel as well. It appears to work, as an oom error that wasn't being reported (thanks to the snippet in #439) would be reported with the bootstrapper in place for a job that was fabricated to cause an oom error.

**Describe the solution you'd like**
If this usage is considered acceptable or expected, it would be nice to have that documented in the docs to bring that functionality to more people. If it's not acceptable/expected, then perhaps just a note that it isn't would be cool in the docs just so no one else goes down this path as I have.

**Comment by xander-jones** [comment]
Hey @ragingdave, no reason why it shouldn't be usable outside of HTTP context, totally acceptable use case. We'll take a look into documenting this 😊.

**Comment by ragingdave** [comment]
@xander-jones I believe I might have found an issue with having that at the console kernel level. When I have that OomBootstrapper in place in the Console bootstrappers, my unit tests will return an exit code of 255. I can also confirm that it's related to the OomBootstrapper as even just commenting out the "meat" of the shutdown function in the OomBootstrapper eliminates the 255 exit code.

aka doing the following allows unit tests to return exit code 0 rather than 255:
```
    public function bootstrap()
    {
        $this->reservedMemory = str_repeat(' ', 1024 * 256);

        register_shutdown_function(function () {
            $this->reservedMemory = null;

            $lastError = error_get_last();

            // If this is an OOM and memory increase is enabled, bump the memory
            // limit so we can report it
//            if ($lastError !== null
//                && app('bugsnag')->getMemoryLimitIncrease() !== null
//                && preg_match($this->oomRegex, $lastError['message'], $matches) === 1
//            ) {
//                $currentMemoryLimit = (int) $matches[1];
//
//                ini_set('memory_limit', $currentMemoryLimit + app('bugsnag')->getMemoryLimitIncrease());
//            }
        });
    }
}
```

I did some more specific testing, and uncommenting/commenting the line:
```
&& app('bugsnag')->getMemoryLimitIncrease() !== null
``` 
is what's triggering the issue. I'm not sure if resolving from the container is the issue or not, but it definitely causes a 255 during unit tests

**Comment by mattdyoung** [comment]
Hi @ragingdave 

We think you're right about not having the full container in your unit tests, so `bugsnag` doesn't exist which will lead to the crash.

We're planning to fix by checking if `bugsnag` exists in the container before trying to use it.

**Comment by yousif-bugsnag** [comment]
This has been released in [v2.22.1](https://github.com/bugsnag/bugsnag-laravel/releases/tag/v2.22.1).
